akka-docs/rst/java/stream/stream-customize.rst

```diff
@@ -50,7 +50,7 @@ override ``onPull()`` which indicates that we are free to emit a single element.
 ``onDownstreamFinish()`` which is called if the downstream cancelled. Since the default behavior of that callback is
 to stop the stage, we don't need to override it. In the ``onPull`` callback we simply emit the next number.
 
-Instances of the above :class:`GraphStage` are subclasses of ``Graph<SourceShape<Int>,Unit>`` which means
+Instances of the above :class:`GraphStage` are subclasses of ``Graph<SourceShape<Int>,NotUsed>`` which means
```

Comment:
It's not as simple as a global grep though - we have one or two places where we indeed leaked a Unit and can't change it because binary compat. Could you sanity check it's consistent with actual code?